src/runtime_src/core/tools/xbmgmt2/OO_LoadConfig.cpp

```diff
@@ -126,5 +128,44 @@ OO_LoadConfig::execute(const SubCmdOptions& _options) const
     return;
   }
 
-  //TO_DO: parse the INI file
-}
\ No newline at end of file
+  try {
+    for (auto& t : deviceCollection)  
+      loadconfig(t);
+  } catch (const std::runtime_error& e) {
+    // Catch only the exceptions that we have generated earlier
+    std::cerr << boost::format("ERROR: %s\n") % e.what();
+    return;
+  }
+}
+
+/*
+ * so far, we only support the following configs
+ * [Device]
+ * mailbox_channel_disable = xxx
+ * mailbox_channel_switch = xxx
+ * cahce_xclbin = xxx
+ */
+void
+OO_LoadConfig::loadconfig(const std::shared_ptr<xrt_core::device>& _dev) const

```

Comment:
This shouldn't be a member funciton. Please make it static const

Reply:
loadconfig -> load_config for readability

Reply:
will change